app/model.js

```diff
@@ -21,7 +21,7 @@ const fetchArticles = () =>
         (SELECT COUNT(*) FROM comments WHERE comments.article_id = articles.article_id)
         AS comment_count
         FROM articles
-        JOIN comments
+        RIGHT JOIN comments
```

Comment:
Right join would only join the articles with comments, we want to join all articles even if they don't have comments, use LEFT JOIN